junit-jupiter-params/src/main/java/org/junit/jupiter/params/provider/MethodSource.java

```diff
@@ -109,7 +109,7 @@
 	 *
 	 * <p>Factory methods in external classes must be referenced by <em>fully
 	 * qualified method name</em> &mdash; for example,
-	 * {@code com.example.StringsProviders#blankStrings}.
```

Comment:
This method does not exist

Reply:
This was merely meant as an example for the syntax of "fully qualified method names".